### DISCUSSION TITLE
When will the message count tracker be implemented?

### DISCUSSION THREAD

**Discussion by MystPi** [Looky1173/Incubator#1] [Q&A] [answered]
As of now, Incubator doesn't really have any use since non of its experiments are implemented. I was just curious about when it will be created. 😸

**Comment by Looky1173** [comment, marked as answer]
There's no ETA yet, but I'm currently working on the backend! You can check how it's coming along [here](https://replit.com/@Looky1173/Message-Count-Tracker).

**Reply by MystPi** [reply]
Good to know; thanks!
